.github/workflows/specimin-evaluation_CI_with_Gradle.yml

```diff
@@ -0,0 +1,66 @@
+name: specimin-evaluation_CI_with_Gradle
```

Comment:
Why are there two near-copies of this file? Only one of them seems to be running (there is only one new CI job); which is it?